linux_docker_resources/entry_point.sh

```diff
@@ -12,41 +12,49 @@ export ORIG_GID=$(echo $ORIGPASSWD | cut -f4 -d:)
 export UID=${UID:=$ORIG_UID}
 export GID=${GID:=$ORIG_GID}
 
+ARCH=`uname -i`
+
 ORIG_HOME=$(echo $ORIGPASSWD | cut -f6 -d:)
 
 echo "Enabling multicast..."
 ifconfig eth0 multicast
 echo "done."
 
-echo "Initializing Git-LFS..."
-if ! sudo -H -u rosbuild -- git lfs install; then
-  echo "Git-LFS failed to initialize (this is expected on ARM platforms)."
+if [ $ARCH = "x86_64" ]; then
+  echo "Initializing Git-LFS..."
+  sudo -H -u rosbuild -- git lfs install
+  echo "done."
+else
+  echo "Not installing git-lfs because we're on a non-x86_64 machine."
```

Comment:
Is it not available for aarch64? I think our use of lfs is not contingent on the architecture.